<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Connect_Port">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Close_port">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>280</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Current Bpm</string>
    </property>
   </widget>
   <widget class="QLabel" name="bpm_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bpm_graph">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>400</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Data</string>
    </property>
   </widget>
   <widget class="QComboBox" name="year_combo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="month_combo">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>370</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="day_combo">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>370</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Month</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Day</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>401</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> bpm avg</string>
    </property>
   </widget>
   <widget class="QLabel" name="bpm_avg_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>401</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> bpm max</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>401</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> bpm min</string>
    </property>
   </widget>
   <widget class="QLabel" name="bpm_max_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>430</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bpm_min_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>578</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
